kernel_gateway/notebook_http/handlers.py

```diff
@@ -35,6 +35,8 @@ class NotebookAPIHandler(TokenAuthorizationMixin,
     kernel_name
         Kernel spec name used to launch the kernel pool. Identifies the
         language of the source code cells.
```

Comment:
If kernel_language is added, the second sentence in the description of kernel_name is no longer accurate.

Reply:
Yes, if `kernel_language` is decided to be added, then this description should be updated.